src/resmom/mom_hook_func.c

```diff
@@ -3531,7 +3531,7 @@ void reply_hook_bg(job *pjob)
 				 * been deleted already.
 				 */
 				if ((pjob2 = job_alloc()) != NULL) {
-					(void)strncpy(pjob2->ji_qs.ji_jobid, jobid, PBS_MAXSVRJOBID);
+					(void)snprintf(pjob2->ji_qs.ji_jobid, PBS_MAXSVRJOBID, jobid);
```

Comment:
There should never be a "%" in a job ID, but just to be careful.
```suggestion
					(void)snprintf(pjob2->ji_qs.ji_jobid, sizeof(jobid), "%s", jobid);
```

Reply:
Doh!  Thanks!  Please take a look now.